corehq/apps/hqadmin/service_checks.py

```diff
@@ -222,6 +223,42 @@ def check_formplayer():
         return ServiceStatus(res.ok, msg)
 
 
+def check_pillowtop():
+    def changes_behind(pillow_meta):
+        lag = 0
+        for k, offset in pillow_meta['offsets'].items():
+            # seq_format is 'json' for KafkaChangeFeed and 'text' for
+            # ChangeFeed. The former seems to have offsets that are greater
+            # than the corresponding seq, while the latter has a seq value
+            # that's greater than the offset
+            if pillow_meta['seq_format'] == 'json':
+                seq = pillow_meta['seq'].get(k, 0)
+                lag += offset - seq
+            else:
+                seq = pillow_meta['seq']
+                lag += seq - offset
+        return lag
```

Comment:
Any idea what the different offsets mean?  I'm flying a little blind here and mostly guessing

Reply:
`seq` from couch is can't be reliably converted into a number which is why there's often some discrepancy.